### DISCUSSION TITLE
Empty doc page for internal module?

### DISCUSSION THREAD

**Discussion by bobjoe12131** [gleam-lang/gleam#3500] [Questions & support] [answered]
I have a project with a module called `util` with a few private or internal functions. Nothing public.
When i run `gleam docs build`, the `util` module shows up in the docs with nothing in it.
Can i hide the module from the docs?

**Comment by giacomocavalieri** [comment, marked as answer]
Yeah! You can mark that module as internal, that way all its public functions will be hidden from the package interface. To do so you can move `util` under `src/project-name/internal/util` or change the default internal modules in your `gleam.toml`, you can find more info [here](https://gleam.run/writing-gleam/gleam-toml/)

**Reply by bobjoe12131** [reply]
Thank you!
